src/main/java/org/opentripplanner/routing/impl/GraphPathFinder.java

```diff
@@ -188,16 +188,11 @@ public GraphPathFinder(Router router) {
             for (GraphPath path : newPaths) {
```

Comment:
The comment on Line 187 should be updated because we're no longer banning trips, but banning trip sequences.

Reply:
Refactored in 6adec27faa2e744f94cd91e4ae4d337a5c25c7c7